gulp/watch.js

```diff
@@ -114,7 +114,7 @@ module.exports = function (gulp, gutil) {
   });
 
   gulp.task('watch-res-sass', function (cb) {
-    watch()(watchSrc(['res/**/' + server.localSkinCompileVersion() + '/**/*.{scss,properties}']), watchOpts, function (file) {
+      watch()(watchSrc(['res/**/*.{scss,properties}']), watchOpts, function (file) {
```

Comment:
Looks okay, although I am not sure if the local skin compile version is still being used. @dougschroeder  Can you please confirm this is the right change? May be @adamayres you know as well.

Reply:
I think this change is fine. I'm not sure we use `localSkinCompileVersion` in the lithium-sdk (and the services sdk was broken out into a separate project). We might want to think about removing logic related to that all-together.

It looks like it was originally from this in https://github.com/lithiumtech/lithium-sdk/commit/9956bd9ab7812c709a670416b1bcdd0dc90eedb7:

```
watch()(['res/**/*.scss'].concat(gutil.env.watchResIgnore || []), watchOpts, function (file) {
```

Reply:
9956bd9 was actually an important change for core engineering which reduced the amount of files being watched, it did this by only watching the sass files in responsive folder for a specific version. 

The specific version is configurable to core engineering by specifying a `localSkinCompileVersion` in the local `server.conf.json`. In the case this is not set then it will default to `v2-lia16.6`. 

For customers it does not make sense to limit them to just this specific folder, but since this code is unfortunately used by both core engineering and external customers, we need to find a solution that works for both - which means preserving the performance improvement of not watching all sass files AND ensuring that the path to the customer's skin is watched.

I think you could write a new file matcher blob that achieves this goal, it might look something like:

```
 watch()(watchSrc([
  'res/feature/responsivepeak/' + server.localSkinCompileVersion() + '/**/*.{scss,properties}',
  'res/**/!(responsivepeak)/**/*.{scss,properties}'
]),
```

You'd need to test and experiment to ensure this works in both environments.